lib/diagnostic.rb

```diff
@@ -31,31 +31,31 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '#{character} is really a Sith Lord.'
```

Comment:
So close here! Remember that you need double quotes in Ruby to do string interpolation.